test/testcases.ts

```diff
@@ -140,7 +140,7 @@ export const testcases: Record<string, Testcase[]> = {
     {
       name: 'glob with dir',
       src: './dir/bar.txt',
-      dest: '/fixture2/bar.txt'
+      dest: '/fixture2/dir/bar.txt'
```

Comment:
`structured: true` sounds good if we don't want to change the behaviour of `flatten: false`.
I don't know what's the behaviour `rollup-plugin-copy`, but the current implementation here behaves strangely when `src` contains multiple `../../` or when `dest` is empty (as pointed out in #46)

Reply:
The current `flatten: false` behaves exactly the same way with `rollup-plugin-copy` because this part of the code is same.

Would you rename `flatten: false` to `structured: true` and add a small description to the "Differences with rollup-plugin-copy" section in the README?

Reply:
Done. 286fb05